src/frontends/pytorch/src/transforms/aten_stack_list_construct_replacer.cpp

```diff
@@ -0,0 +1,58 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "aten_stack_list_construct_replacer.hpp"
+
+#include "openvino/core/rt_info.hpp"
+#include "openvino/op/util/framework_node.hpp"
+#include "openvino/pass/pattern/matcher.hpp"
+#include "openvino/pass/pattern/op/wrap_type.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace pass {
+
+AtenStackListConstructReplacer::AtenStackListConstructReplacer() {
+    auto stack = ov::pass::pattern::wrap_type<ov::op::util::FrameworkNode>();
+
+    ov::matcher_pass_callback callback = [](ov::pass::pattern::Matcher& m) {
+        auto stack = cast_fw_node(m.get_match_root(), "aten::stack");
+        if (!stack) {
+            return false;
+        }
+        auto input_node = stack->input_value(0).get_node_shared_ptr();
+        auto axis_node = stack->input_value(1).get_node_shared_ptr();
+        auto axis_const = std::dynamic_pointer_cast<opset10::Constant>(axis_node);
+        auto axis = axis_const->cast_vector<int64_t>();
+
+        if (auto list_construct = cast_fw_node(input_node, "prim::ListConstruct")) {
+            auto list_inputs = list_construct->input_values();
+            OutputVector node_vector;
+            auto zero = opset10::Constant::create(element::i32, Shape{}, {0});
+            for (size_t i = 0; i < list_inputs.size(); i++) {
+                auto node = concat_list_construct(list_inputs[i].get_node_shared_ptr());
+                auto unsqueezed_node = std::make_shared<opset10::Unsqueeze>(node, axis_const);
+                node_vector.push_back(unsqueezed_node);
+            }
+            auto concat = std::make_shared<opset10::Concat>(node_vector, axis[0]);
+
+            copy_runtime_info({stack, input_node}, concat);
+            replace_node(stack, concat);
+            return true;
+        }
+        return false;
+    };
+
+    auto m =
+        std::make_shared<ov::pass::pattern::Matcher>(stack,
```

Comment:
Could you please use namespace `ov::pass::pattern` so that this line was shorter.